### PR TITLE
Created the schema

### PR DESCRIPTION
**Pull request by Jakub-Poskrop**
Solves issue #8

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    plant_id INT NOT NULL PRIMARY KEY, 
+    botanist_id INT NOT NULL, 
+    plant_name VARCHAR(100) NOT NULL, 
+    FOREIGN KEY(botanist_id) REFERENCES botanist 
+);
+
+CREATE TABLE recording(
+    recording_id INT NOT NULL PRIMARY KEY, 
+    plant_id INT NOT NULL,
+    soil_moisture DECIMAL(8,2) NOT NULL,
```
Comment (Kurt812) [inline schema.sql:23]: can we limit the values to sit between 0 and 100

File: schema.sql [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    botanist_id INT NOT NULL, 
+    plant_name VARCHAR(100) NOT NULL, 
+    FOREIGN KEY(botanist_id) REFERENCES botanist 
+);
+
+CREATE TABLE recording(
+    recording_id INT NOT NULL PRIMARY KEY, 
+    plant_id INT NOT NULL,
+    soil_moisture DECIMAL(8,2) NOT NULL, 
+    temperature DECIMAL(8,2) NOT NULL,
```
Comment (ebradley12) [inline schema.sql:24]: CHECK (temperature BETWEEN -50 AND 50)

File: schema.sql [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    last_watered TIMESTAMP NOT NULL,
```
Comment (ebradley12) [inline schema.sql:25]: What if the plant hasn't been watered yet?

**Review by ebradley12 — COMMENTED** [review]
Great work!

**Review by Kurt812 — APPROVED** [review]
Nice work!

**Pull request merged**